app/DoctrineMigrations/Version20160712115709.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Class Version20160712115709
+ * @package Application\Migrations
+ */
+class Version20160712115709 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf(
+            $this->connection->getDatabasePlatform()->getName() != 'mysql',
+            'Migration can only be executed safely on \'mysql\'.'
+        );
+
+        $this->addSql('ALTER TABLE Subscription ADD environment VARCHAR(255) NOT NULL');
```

Comment:
Fixed, though we'll likely just install a new version and import the subscriptions...